### ISSUE TITLE
Allow Waterlogging of Blocks

### ISSUE THREAD

**Issue by WenXin20** [MysticMods/Firefly#1]
**Describe your suggestion:**

For 1.14.+, blocks should be able to waterlog. This will allow for more decoration options and immersion.

**Comment by VulpTheKangaFox** [comment]
While I can agree that _some_ blocks might be better waterlogged, not every single block requires itself to waterlog. Even vanilla doesn't play by those rules with most of their blocks.

**Comment by WenXin20** [comment]
True, though I still think that it would still be nice to see. But up to you.

**Comment by Martacus** [comment]
I will take a look at this, it won't be priority
